modules/openapi-generator/src/main/resources/typescript-angular/modelGeneric.mustache

```diff
@@ -1,10 +1,6 @@
 export interface {{classname}}{{#allParents}}{{#-first}} extends {{/-first}}{{{.}}}{{^-last}}, {{/-last}}{{/allParents}} { {{>modelGenericAdditionalProperties}}
 {{#vars}}
-    {{#description}}
-    /**
-     * {{{.}}}
-     */
-    {{/description}}
+    /** {{#deprecated}}@deprecated {{/deprecated}}{{#description}}{{{.}}} {{/description}}*/
```

Comment:
lets do something like
```suggestion
    {{#description}}
    /** 
    {{#deprecated}}
    * @deprecated
    {{/deprecated}}
    * {{{description}}}
    */
    {{/description}}
    {{^description}}
    {{#deprecated}}
    /** @deprecated */
    {{/deprecated}}
    {{/description}}
```

this avoids empty docs and makes it more compact if its deprecated but there is no description

Reply:
That's nice. I think we should but the `deprecated` tag on the line below `description` in that case if you agree.
It's more common to have description above the tags.

Source: 
- https://jsdoc.app/about-getting-started
- https://tsdoc.org

Reply:
sounds good

Reply:
Updated the PR, and the PR is much cleaner/smaller due to this change too 👍